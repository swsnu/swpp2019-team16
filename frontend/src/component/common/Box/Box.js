import React from 'react';
import PropTypes from 'prop-types';
import Box from '@material-ui/core/Box';

BoxMaterial.propTypes = {
  children: PropTypes.node.isRequired,
};

function BoxMaterial({
  children,
}) {
  return (
    <Box>
      {children}
    </Box>
  );
}

export default BoxMaterial;
