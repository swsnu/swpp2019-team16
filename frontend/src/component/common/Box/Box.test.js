import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Box from './';

describe('<Box />', () => {
  it('SHOULD match with snapshot', async () => {
    const { container } = render(<Box children={<div>Box Text</div>} />);

    expect(container).toMatchSnapshot();
  });
});
