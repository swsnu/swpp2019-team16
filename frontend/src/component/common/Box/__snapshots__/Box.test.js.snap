// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> SHOULD match with snapshot 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    <div>
      Box Text
    </div>
  </div>
</div>
`;
